docs/sources/mimir/configure/configure-spread-minimizing-tokens/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+aliases:
+  - ../operators-guide/configure/configure-spread-minimizing-tokens/
+description: Learn how to migrate ingesters to spread-minimizing tokens.
+weight: 115
+menuTitle: Spread-minimizing tokens
+title: Migrate ingesters to spread-minimizing tokens
+---
+
+# Migrate ingesters to spread-minimizing tokens
+
+Using this guide, you can configure Mimir's ingesters to use the _spread-minimizing token generation strategy_ .
+
+## Before you begin
+
+The ingester time series replication should be [configured with enabled zone-awareness](https://grafana.com/docs/mimir/latest/configure/configure-zone-aware-replication/#configuring-ingester-time-series-replication).
+
+{{% admonition type="note" %}}Use spread-mimizing tokens are recommended if [shuffle sharding](https://grafana.com/docs/mimir/latest/configure/configure-shuffle-sharding/#ingesters-shuffle-sharding) is disabled in your ingesters, or, if shuffle-sharding is enabled, but most of the tenants of your system use all available ingesters.
```

Comment:
I replaced the notes with the admonition shortcode in your document.  

Notes, tips, warnings, and caution admonitions appear with special formatted boxes when rendered. The [admonition shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/) tells Hugo to set this paragraph apart with different formatting. 

Here's what the above admonition shortcode looks like in the doc preview: 
![image](https://github.com/grafana/mimir/assets/104772500/c1c78a80-e107-4798-9a54-b95e2f504ea5)

Reply:
and I found another mistake in this sentence. Corrected below. 
```suggestion
{{% admonition type="note" %}}Spread-mimizing tokens are recommended if [shuffle sharding](https://grafana.com/docs/mimir/latest/configure/configure-shuffle-sharding/#ingesters-shuffle-sharding) is disabled in your ingesters, or, if shuffle-sharding is enabled, but most of the tenants of your system use all available ingesters.
```